user_doc/db_recovery.md

```diff
@@ -55,7 +109,32 @@ If the Packages file now passes the verify step, then as an additional sanity ch
 If this query passes without generating any messages to standard error, then it is time to rebuild the indexes by invoking:
 
 ```
-# rpm -v --rebuilddb
+# rpmdb -v --rebuilddb
+```
+
+At this point you should have a functioning RPM database again.
+
+
+# If all fails
+
+
+If the recovery failed, then a bug should be reported. Note that this is a rare event and it is possible that your system took more damage than just the rpm database.
+
+When creating the bug report, provide the tar.gz backup of the RPM DB as an attachment. If available also provide the list of installed packages. First try
+
+```
+# rpm -qa 1> /dev/null
+```
+
+If this does not work or the result is suspiciously short you can look into `/var/log/rpm*`. RPM comes with a cron script to back up the list of installed packages daily there but it may not be installed on the system.
+
+Another place to look for the list of installed packages may be the logs of a higher level installer/updater that may still be undamaged.
+
+If you have the list of packages available it may be easier to re-create the rpmdb than reparing it. Run
+
+```
+rpm --initdb
```

Comment:
Should be "rpmdb --initdb", but the step is unnecessary: it wont do anything to an existing database, and is not needed to prepare for installation either.